_posts/en/newsletters/2021-11-17-newsletter.md

```diff
@@ -0,0 +1,154 @@
+
+- **Muun wallet supports taproot:**
+  Muun wallet enabled taproot address support after activation occurred,
+  including the ability for users to default to taproot receive addresses.
+
+- **Lightning-based social feed squeaknode launches:**
+  [Squeaknode][squeaknode github] is a Twitter-like P2P network based on LN
+  (using LND). Users can create, view, buy, and sell messages known as "squeaks".
+
+- **Kollider launches alpha LN-based trading platform:**
+  Kollider's latest [announcement][kollider blog] details the derivative
+  platform's features including LN deposits and withdrawals, LNAUTH and LNURL
+  support, and a LN-based referral scheme.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.14.0-beta.rc3][] is a release candidate that includes
+  additional [eclipse attack][topic eclipse attacks] protection (see
+  [Newsletter #164][news164 ping]), remote database support ([Newsletter
+  #157][news157 db]), faster pathfinding ([Newsletter #170][news170
+  path]), improvements for users of Lightning Pool ([Newsletter
+  #172][news172 pool]), and reusable [AMP][topic amp] invoices
+  ([Newsletter #173][news173 amp]) in addition to many other features
+  and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #22934][] adds a verification step after both ECDSA signatures and
+  [schnorr signatures][topic schnorr signatures] are created.  This may
+  prevent the software from disclosing an incorrectly-generated
+  signature that may leak information about the private key or nonce
+  used to generate it.  This follows advice given in a previous update
+  to [BIP340][] (see [Newsletter #87][news87 bips886]) previously discussed in
+  [Newsletter #83][news83 safety].
+
+- [Bitcoin Core #23077][] enables address relay via CJDNS, making CJDNS
+  a fully supported network like IPv4, IPv6, Tor, and I2P. Once CJDNS is
+  set up outside of Bitcoin Core, node operators can toggle the new
+  configuration option `-cjdnsreachable` to have Bitcoin Core interpret
+  `fc00::/8` addresses as belonging to CJDNS rather than being
+  interpreted as private IPv6 addresses.
+
+<!-- FIXME: harding to add topic for onion messages -->
+- [Eclair #1957][] adds basic support for [onion messages][news92 onion
+  messages] per [BOLTs #759][], allowing relaying of onion messages, but
```

Comment:
Would it be better to link to http://localhost:4000/en/newsletters/2020/02/26/#ln-direct-messages for onion messages (which is the description of the proposal, rather than the c-lightning implementation)?

Reply:
This is resolved by the new topic addition for onion messages.